concepts-wip/bools/about.md

```diff
@@ -0,0 +1,144 @@
+# Bools
+
+Crystal has a type know as [`Bool`][bools], it is used to represent the values `true` and `false`.
+With the bool type can you use logical operators(`!`, `||`, `&&`) which can be used to combine bools and change their values.
+
+```crystal
+true && true
+# => True
+
+true && false
+# => False
+
+false && false
+# => false
+
+false || true
+# => True
+false || false
+# => false
+
+!false
+# => true
+
+!true
+# => false
+```
+
+## Logical operators
+
+Crystal has 3 logical operators which is used to combine bools and change their values.
+
+### And(`&&`)
+
+The [_and_ operator][and] in crystal is represented by `&&` and returns `true` if both values given is `true` otherwise it returns `false`.
+When using the _and_ operator so should one bool be placed one the right side of the `&&` and one on the left side.
+
+```crystal
+true && true
+# => true
+
+true && false
+# => false
+
+false && false
+# => false
+```
+
+### Or(`||`)
+
+The [_or_ operator][or] in crystal is represented by `||` and returns `true` if **at least one** of values given is `true` if both of the values are `false` then it returns `false`.
+When using the _or_ operator so should one bool be placed one the right side of the `||` and one on the left side.
+
+```crystal
+true || true
+# => true
+
+true || false
+# => true
+
+false || true
+# => true
+
+false || false
+# => false
+```
+
+### Not(`!`)
+
+The _not_ operator in crystal is represented by `!` and returns `true` if the given bool is `false` and return `false` if `true` is given.
+When using the _not_ operator so should the bool be placed after the operator (`!`).
+
+```crystal
+!true
+# => false
+
+!false
+# => true
+```
+
+## Using brackets(`()`)
```

Comment:
You might want to use the word "parentheses" everywhere (singular: parenthesis, plural: parentheses). Technically parentheses are a type of brackets, but parentheses is a more specific term that most people use to describe those characters `()`